tools/check-capitalization

```diff
@@ -16,7 +16,7 @@ from scripts.lib.zulip_tools import WARNING, FAIL, ENDC
 
 from tools.lib.capitalization import check_capitalization
 
-DJANGO_PO_REGEX = re.compile('msgid "(.*?)"')
+DJANGO_PO_REGEX = re.compile('message_id "(.*?)"')
```

Comment:
This is not our name to change, it’s built into the `.po` file format. We need to exclude this.

Reply:
Ok, one more question, should I only exclude those two files, or the entire tools directory?

Reply:
I’d say exclude the two files.